05/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/05/css/global.css" />
+    <link
+      rel="stylesheet"
+      media="(max-width: 799px)"
+      href="/05/css/mobile.css"
+    />
+    <link
+      rel="stylesheet"
+      media="(min-width: 800px)"
+      href="/05/css/desktop.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <section class="container nav__container">
+        <nav class="nav"></nav>
+        <div class="nav__two"></div>
+        <button class="hamburger"></button>
+      </section>
+    </header>
+    <section class="section__two">
+      <section class="container articles__container articles">
+        <div class="article article--one"></div>
+        <div class="article article--two"></div>
+      </section>
+    </section>
+    <section class="section__three">
```

Comment:
Jeśli mamy `element` tj. `section__three` to w strukturze powinien występować `block` tj. `section`

Reply:
> 
chyba nie zrozumiałam :)

Reply:
Chodzi o to, że jeśli masz gdzieś z nazwę z BEM typu `section__three` to gdzięś w strukturze musisz mieć klasę `section` czyli (znaczniki tutaj nie są istotne):
```
<!-- zle -->
<div> 
  <p>bla bla bla</p>
  <div class="section__three"></div>
</div>
```

```
<!-- dobrze -->
<div class="section"> 
  <p>bla bla bla</p>
  <div class="section__three"></div>
</div>
```
Teraz jest to jasne?